CFC_DataCollector/recommender/get_trips.py

```diff
@@ -0,0 +1,29 @@
+from userclient import getClientSpecificQueryFilter
+from get_database import get_section_db
+
+class TripFactory:
```

Comment:
I am not sure if I would call this a factory. A factory typically takes an input and returns the appropriate subclass. This seems more like an iterator.